### PR TITLE
fix: add tsfamily dead lock

### PR DESCRIPTION
**Pull request by Subsegment**
# Required checklist
- [ ] Sample config files updated (`config`,`meta/config` and `default config`) 
- [ ] If there are user-facing changes, the documentation needs to be updated prior to approving the PR( [Link]() )
- [ ] If there are any breaking changes to public APIs, please add the `api change` label.
- [ ] Signed [CLA](https://cla-assistant.io/cnosdb/cnosdb) (if not already signed)

# Which issue does this PR close?

[//]: # (We generally require a GitHub issue to be filed for all bug fixes and enhancements and this helps us generate change logs for our releases. You can link an issue to this PR using the GitHub syntax. For -- example `Closes #123` indicates that this PR will close issue #123.)

Related #.

# Rationale for this change

[//]: # (Why are you proposing this change? If this is already explained clearly in the issue then this section is not needed.
 Explaining clearly why changes are proposed helps reviewers understand your changes and offer better suggestions for fixes.)

# Are there any user-facing changes?

[//]: # (There is no need to duplicate the description in the issue here, but it is sometimes worth providing a summary of the individual changes in this PR.)

### CODE REVIEW

File: config/config.toml [hunk 1]
```diff
@@ -90,14 +90,18 @@ path = '/var/lib/cnosdb/wal'
...
 
-## The maximum amount of immutable caches.
-#max_immutable_number = 4
+# The maximum amount of immutable caches.
+max_immutable_number = 4
+
+# The partion number of memcache cache,default equal to cpu number
+partition = 16
+
+# The maximum size of database usage schema buffer
```
Comment (zipper-meng) [inline config/config.toml:102]: - Does `database usage schema` means identity `database USAGE_SCHEMA`?
- The `size` is a property of the buffer of the `database usage schema`

File: config/config.toml [hunk 2]
```diff
@@ -90,14 +90,18 @@ path = '/var/lib/cnosdb/wal'
 #sync_interval = "0" # h, m, s
 
 [cache]
-## The maximum size of a mutable cache.
-#max_buffer_size = "128M" # 134,217,728 bytes
+# The maximum size of a mutable cache.
+max_buffer_size = "128M" # 134,217,728 bytes
```
[the remark this answers is not in the document]
Comment (Subsegment) [reply config/config.toml:94]: Why keep those configurations being commented.

**Review by zipper-meng — APPROVED** [review]
It's ok

**Comment by zipper-meng** [comment]
Sending a message to a bounded channel while holding a write lock would cause a dead lock, this PR fixed a representation of the issue.

**Pull request merged**